dfc/target.go

```diff
@@ -426,7 +426,8 @@ existslocally:
 	written, err := io.CopyBuffer(w, file, buf)
 	if err != nil {
 		errstr = fmt.Sprintf("Failed to send file %s, err: %v", fqn, err)
-		t.invalmsghdlr(w, r, errstr)
+		glog.Errorln(t.richHTTPError(r, errstr, http.StatusInternalServerError, 1))
```

Comment:
Just an idea to discuss: what if we remove skip from arguments and use a constant skip=2?
1. No need to pass 'skip' - except this line 2 is used
2. As for this line, even with skip=2 we have a) target.go:429 in log because glog ouputs its line and b) at the same time, we will know who called 'existslocally' because of skip=2 show it - that can be sometimes more useful than to know that utility function 'existslocally' failed

Reply:
It would work fine for now, but it may have to be increased at some point in the future. The thing I don't like is that it is not explicit - users of the function would have to remember that the function _only_ skips two stack frames.

Reply:
Also, the file and line # appended to the end of the error string will always be the caller from two stack frames up. This could be misleading at times.